plugins/modules/dcnm_interface.py

```diff
@@ -422,7 +422,7 @@
             - When ethernet interface is a PortChannel or vPC member, mode is ignored.
               The only properties that can be managed for PortChannel or vPC member interfaces
               are 'admin_state', 'description' and 'cmds'. All other properties are ignored.
-            choices: ['trunk', 'access', 'routed', 'monitor', 'epl_routed']
+            choices: ['trunk', 'access', 'routed', 'monitor', 'epl_routed', 'dot1q_tunnel_host']
```

Comment:
This is an informational note only for future reference.  I added support for vcp_dot1q_tunnel_members that are part of a port-channel [here](https://github.com/CiscoDevNet/ansible-dcnm/pull/382/files#diff-08575ad0f483bf55d1059fb810be34592862f247ea7d838fb7acfaa782b0f21fR1849)